src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="url"
+      data-qa="big"
+      method="post"
+      class="container"
+    >
+      <label
+        class="big-search search"
+        data-qa="big"
+        >
+        <img src="../src/images/Search.svg" alt="big-search-logo">
```

Comment:
There is no need to use '-' in alt attribute, also it is unnecessary for user read if it's big search logo or small, especially for users with disability, 'Search logo' is enough